docs/features.rst

```diff
@@ -2319,6 +2319,37 @@ The stage ``{"$match": { "name": "$name", "time": "$time"}}`` in the pipeline wi
 The request above will ignore ``"count": {"$sum": "$value"}}``. A
 Custom callback functions can be attached to the ``before_aggregation`` and ``after_aggregation`` event hooks. For more information, see :ref:`aggregation_hooks`.
 
+# Special Note on PATCH
+~~~~~~~~~~~
+``PATCH`` **cannot** remove a field but only update value of the field.
+
+Consider the following schema:
+
+```
+'entity': {
+  'name': {
+    'type': 'string',
+    'required': True },
+  'contact': {
+    'type': 'dict',
+    'required': True,
+    'schema': {
+      'phone': {
+        'type': 'string',
+        'required': False,
+        'default': '1234567890' },
+      'email': {
+        'type': 'string',
+        'required': False,
+        'default': 'abc@efg.com' },
+    }
+  }
+}
+```
+
+Two notations ``contact: { email: 'an email'}`` and ``contact.email: 'an email'`` can be used to update the `email` field embedded in `contact` field.
+
+
```

Comment:
Since we're adding a special noe on PATCH, I feel that this is the right place to explain what's the purpose of the new setting (basically, give an example like the one in the original ticket).